### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by morenol**
Fixes #

### Checklist
- [ ] My branch is up-to-date with development branch.
- [ ] Everything works and tested on latest stable Rust.
- [ ] Coverage and Linting have been applied

### CODE REVIEW

**Comment by Mec-iS** [comment]
looks like `x86_64-pc-windows-msvc` doesn't like Rust 1.65

In particular `libm`, I reached out to them; https://github.com/rust-lang/libm/issues/271

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -52,11 +54,14 @@ jobs:
...
           args: --all-features --target ${{ matrix.platform.target }}
       - name: Tests
-        if: matrix.platform.target == 'x86_64-unknown-linux-gnu' || matrix.platform.target == 'x86_64-pc-windows-msvc' || matrix.platform.target == 'aarch64-apple-darwin'
+        if: matrix.platform.target == 'x86_64-unknown-linux-gnu' || matrix.platform.target == 'aarch64-apple-darwin'
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: --all-features
+      - shell: wsl-bash {0}
+        run: cargo test --all-features
```
Comment (Mec-iS) [inline .github/workflows/ci.yml:63]: please let's stick to the convention of having the `if` before the `run`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -33,6 +33,8 @@ jobs:
             ./target
           key: ${{ runner.os }}-cargo-${{ matrix.platform.target }}-${{ hashFiles('**/Cargo.toml') }}
           restore-keys: ${{ runner.os }}-cargo-${{ matrix.platform.target }}-${{ hashFiles('**/Cargo.toml') }}
+      - uses: Vampire/setup-wsl@v1
```
Comment (Mec-iS) [inline .github/workflows/ci.yml:36]: I suppose we need this for `686-pc-windows-msvc`?

Comment (morenol) [reply .github/workflows/ci.yml:36]: I saw this workaround to our issue: https://github.com/dudykr/stc/blob/64bf11b206d5e93ea9901022ab91273574f61c28/CONTRIBUTING.md#troubleshooting

Let me try to install `cargo` here and see if this fixes everything

**Comment by morenol** [comment]
> looks like `x86_64-pc-windows-msvc` doesn't like Rust 1.65 (but works with `686`)


for i686 we don't run the tests (github runners are x86_64) , just stable build, probably there is a test dependency that fails, hopefully this WSL stuff will solve that issue

File: Cargo.toml [hunk 3]
```diff
@@ -49,11 +49,9 @@ resolver = "2"
 [profile.test]
 debug = 1
 opt-level = 3
-split-debuginfo = "unpacked"
```
Comment (morenol) [inline Cargo.toml:52]: looks like issue was here @Mec-iS

Comment (Mec-iS) [reply Cargo.toml:52]: cool. i think Rust 1.65 dropped support for that

**Review by Mec-iS — APPROVED** [review]

**Pull request merged**